frontend/src/utils/verified-models.ts

```diff
@@ -1,6 +1,6 @@
 // Here are the list of verified models and providers that we know work well with OpenHands.
 export const VERIFIED_PROVIDERS = ["openai", "azure", "anthropic"];
-export const VERIFIED_MODELS = ["gpt-4o", "claude-3-5-sonnet-20240620"];
+export const VERIFIED_MODELS = ["gpt-4o", "claude-3-5-sonnet-20241022"];
```

Comment:
We can probably keep the old Sonnet here, if this is _the list of verified models that we know work well_... it did, didn't it? 🤔

Reply:
Agree here

Reply:
It also make sense to me that we only put the new sonnet as verified model to encourage ppl using it 😓

Reply:
I don't know... We already "recommend" somewhere else, I know we also have in addition a little list where we add models as we verify them to be decent...

Reply:
I mean, these are two different things, at least according to the current documentation. We have:
- one recommended model
- verified models
- the rest - all should work (start, run), but we tell people we don't know how well, and don't recommend them... 

Does that make sense?